### PR TITLE
Update third party notices

### PR DESCRIPTION
**Pull request by carlonewrelic**
## Proposed Release Notes
Updated third party notices

## Links
#405 

## Details
Updated third party notices.  I had to manually add the Symantec Certificate license to the third party notices file.

### CODE REVIEW

File: THIRD_PARTY_NOTICES.md [hunk 1]
```diff
@@ -856,12 +2057,4 @@ contractors. Neither party nor its employees, consultants, contractors, or agent
 or joint venturers of the other party, nor do they have any authority to bind the other party by contract or
 otherwise to any obligation.
 Root Certificate License Agreement v3.0 (January 2014)
-```
-
-Broadcom Copyright Statement (https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/symantec-security-software/web-and-network-security/ssl-visibility/4-5/copyright-statement-v135048856.html)
```
Comment (carlonewrelic) [inline THIRD_PARTY_NOTICES.md:861]: Removed the Broadcome Copyright statement based on guidance from legal.

File: THIRD_PARTY_NOTICES.md [hunk 2]
```diff
@@ -7,26 +7,461 @@ below.
...
 * [async](#async)
-* [concat-stream](#concat-steam)
+* [concat-stream](#concat-stream)
 * [https-proxy-agent](#https-proxy-agent)
 * [json-stringify-safe](#json-stringify-safe)
 * [readable-stream](#readable-stream)
 * [semver](#semver)
-* [@grpc/grpc-js](#@grpc/grpc-js)
-* [@grpc/proto-loader](#@grpc/proto-loader)
-* [Symantec](#Symantec)
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES.md:23]: may still want the symantec in the contents

File: THIRD_PARTY_NOTICES.md [hunk 3]
```diff
@@ -7,26 +7,461 @@ below.
...
+* [@newrelic/superagent](#newrelicsuperagent)
+* [@tyriar/fibonacci-heap](#tyriarfibonacci-heap)
 * [async](#async)
-* [concat-stream](#concat-steam)
+* [concat-stream](#concat-stream)
 * [https-proxy-agent](#https-proxy-agent)
 * [json-stringify-safe](#json-stringify-safe)
 * [readable-stream](#readable-stream)
 * [semver](#semver)
-* [@grpc/grpc-js](#@grpc/grpc-js)
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES.md:21]: i see proto-loader got moved above but i don't see grpc-js? you might double-check your package-lock is in good shape.

File: THIRD_PARTY_NOTICES_ADDENDUM.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
+## Third Party License Addendum
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES_ADDENDUM.md:1]: You have both a third party licenses and a third party notices addendum. I don't think you need both of these.

Comment (carlonewrelic) [reply THIRD_PARTY_NOTICES_ADDENDUM.md:1]: 👍 just deleted the other THIRD_PARTY_LICENSES_ADDENDUM.md file

File: THIRD_PARTY_NOTICES.md (hunk 2 above)
Comment (carlonewrelic) [reply THIRD_PARTY_NOTICES.md:23]: I moved it below to and 'additional-licenses' section at line 67

**Pull request merged**